src/tools/dev/scripts/visit-build-open

```diff
@@ -380,9 +380,9 @@ echo "The database plugins:"                  >> resultlog 2>&1
 ls $dist/build/plugins/databases/libI* | sed "s/$dist\/build\/plugins\/databases\/libI//" | sed "s/Database.so//" >> resultlog 2>&1
 EOF
 
-if [ $kickit = true ]
+if [ "$kickit" = "true" ]
```

Comment:
how come this `"true"` is in quotes but the ones at the start are not? How does bash work

Reply:
IMHO, the quote form is a tad "safer". Here's why...

```
sh-3.2$ if [ $foo = bar ]; then echo "got here"; fi
sh: [: =: unary operator expected
sh-3.2$ if [ "$foo" = "bar" ]; then echo "got here"; fi
```

In the second form, its just a harmless empty string compared with `"bar"`. In the first form, its trying to interpret the operands as other than strings.